datashader/tests/conftest.py

```diff
@@ -0,0 +1,34 @@
+CUSTOM_MARKS = ("benchmark",)
```

Comment:
Copied from HoloViews. Basically, it makes it so `benchmark` only runs when you pass in `--benchmark`. Benchmark does not work with `pytest-xdist` (it emits a warning if both are enabled at the same time, which is why I disable it with `--benchmark-skip`.

At some point, I would also like to add a GPU marker, so it is possible to run only the GPU tests and not all the tests with GPU tests enabled.